tests/test_utilities.py

```diff
@@ -64,7 +64,7 @@ def make_uniform_pop_data(age_bin_midpoint=False):
     age_bins = [(b.age_start, b.age_end) for b in make_age_bins().itertuples()]
     sexes = ("Male", "Female")
     years = zip(range(1990, 2018), range(1991, 2019))
-    locations = (1, 2)
+    locations = list(range(1, 2))
```

Comment:
This feels contrived. Are you aware `list(range(1, 2))` == [1]. Is this intended? If so, why not just [1]?

Reply:
Yes - this was causing issues with population creation. It was creating a population that had both population 1 and 2 and the index for all sex age groups for both locations. But then when we create the population we were just setting location to 1 so we had duplicate index.